freiburg/services.md

```diff
@@ -17,47 +17,75 @@ The Freiburg Galaxy Team is offering several services to enable **reproducible**
+With **free** registration to the [UseGalaxy.eu](https://usegalaxy.eu) server we provide you an **easy access** to:
 
-When this effort is combined with our community-maintained workflows and our in-depth training material, it makes up for a truly productive work experience. We believe in enabling everyone to perform reproducible science.
+- Free compute and storage resources (250 GB per user)
+- Databases (e.g. EMBL-EBI ENA, NCBI SRA, Ensembl, UCSC, UniProt, and many organism-specific databases)
+- More than 2000 different, well-documented and constantly maintained bioinformatics tools
+- Puplic HTS data analyis workflows
+- Reference genomes
+- Visualizations
+
+When this effort is combined with our community-maintained workflows and our in-depth [Galaxy training material](https://training.galaxyproject.org), it makes up for a truly productive work experience. We believe in enabling everyone to perform reproducible science.
+
+Simply upload your data from your computer or via FTP and get started with your data analysis!
+
+### Galaxy Subcommunities
+
+We want to encourage scientist to join forces and foster subcommunities by giving them a common place, a Galaxy subdomain page. Every sububdomains comes with own welcome page, specific tool box, example data, workflow, and training material. More information can be found on the [https://galaxyproject.eu](https://galaxyproject.eu/) webpage.
 
 ---
 <a name="training"></a>
 # <i class="fa fa-graduation-cap"></i> Training
 
-Supporting researchers by educating them in *e.g.* big data analysis, programming, data management, and Galaxy server administration, is one of our goals as we believe that **sharing of knowledge** and the **open science** movement are the key points in future.
+We are passionate about training! Our team wants to support researchers to take part in their own data analyses by educating them in *e.g.* big data analysis, programming, data management, and Galaxy server administration. We believe that **sharing of knowledge** and the **open science** movement are the key points in future.
 
 <div class="multiple-img" style="text-align: right;">
     <img src="/assets/media/training_1.jpg" width="200px" alt="Workshop picture" />
     <img src="/assets/media/training_2.jpg" width="200px" alt="Workshop picture" />
     <img src="/assets/media/training_3.jpg" width="200px" alt="Workshop picture" />
 </div>
 
-### Workshops
+### Galaxy Training Workshops
+
+Locally in Freiburg, we offer twice per year a free full-week hands-on high-throughput sequencing data analysis workshop. A typical workshop schedule looks as follows: 
+
+- Monday:     Introduction to Galaxy and HTS
+- Tuesday:    Quality control, IGV, ChIP-Seq, 
+- Wednesday:  RNA-Seq 
+- Thursday:   HiC data analysis, Methyl-C data analysis
+- Friday:     Bring your own data, exercises
+
+Topice usually are selected from desired topics of the applicants. 
```

Comment:
Topics?

Reply:
Take
"We have more training material for various topics to teach, e.g. Transcriptomics, Metabolomics, Metagenomics, Proteomics, Genome Annotation, Variant Calling, ... . Topics usually are selected from desired topics of the applicants.

Reply:
I mean there is a spelling mistake I think.